LamestWebserver/LamestWebserver/Core/Logger.cs

```diff
@@ -0,0 +1,268 @@
+﻿using System;
+
+namespace LamestWebserver.Core
+{
+    /// <summary>
+    /// A simple Logger for LamestWebserver.
+    /// </summary>
+    public class Logger
+    {
```

Comment:
add destructor, that calls `Close`.
add constructor, that calls new `protected void Open()`